### DISCUSSION TITLE
Proposal  for Community Governance Scheme

### DISCUSSION THREAD

**Discussion by sxxfuture-official** [filecoin-project/FIPs#945] [Enhancements - Governance]
# Proposer

SXX Future Data from Asia Storage Provider Working Group

# Background

The Filecoin network has reached a critical moment. Since November last year, the storage power (raw byte) of the whole network has been declining, and storage nodes (some with verified data) have been terminated. This threatens the network security and the security of humanity’s data. The progress of FEVM, IPC and retrieval are significantly slower than expected than the original economic timeline.

In 2024, we are seeing an upsurge of crypto assets value, and many new projects have landed on various exchanges around the world. Inscriptions, BRC20 and Ethereum 2.0 have been warmly discussed again. However, the price performance of Filecoin is stagnating which reflects it’s value, the morale of the community is low, year-on-year the survival rate of SP continues to drop, institutional investors withhold investment on Filecoin or give up, active Filecoin developers are reducing, and the relocation of various social attention away from Filecoin is now a trend. If Filecoin continues this way, the whole network will be dissolved soon.

Everyone in the Filecoin community has cherished the mission of Fielcoin and technological advancements in its early days, firmly believed in the power of technological progress, believed in Web3.0, and was determined to change the world with distributed storage. This situation is unbearable and hard to let go!

The re-emergence of Filecoin requires the joint efforts of both PL,  FF and the community. PL and FF need to show a sense of responsibility and give the market more confidence; The community should also exert its strength, actively participate in the governance, and work together to promote the healthy and rapid development of the chain and its related technologies.

# Proposal

1. **Protocol Labs and Filecoin Foundation review the current tokenomics model:** Especially the designed mechanism of FIL+. Consult with the community and put forward new reform schemes to ensure the authenticity of Data Caps and promote the healthy development of verified data onboarding.
2. **Pay attention to market value management:** The healthy development of public chains needs more social attention, ecosystem developers, token holders, storage providers, investors and protocol engineers. In a competitive environment, low market value means that the prospects are slim and the pledge is devalued. Low market value destroys market confidence and keeps users away. Only a high market value can spur more investor’s attention. As a Web3.0 storage infrastructure, Filecoin should aim at the top ten in market value. Increase its ranking in crypto indexes such as CoinMarketCap.
3. **Strengthen the communication between the core team and the community:** The work of the core team determines the performance of the chain and the future development direction of the chain. Set up a joint working group, and the community and core team will hold regular meetings to inform the technical development planning and progress, incorporate feedback from the community on the pain points, and effectively advance the technology and meet market demands.
4. **Return the governance back to the community:** Community voting system should be  adopted for major issues. Major issues related to the ecosystem, such as the reform plan of Fil+ and the direction of technological development, are to be decided by community voting not by core member’s discussions.
5. **Working groups rotate representatives/advocates:** For example, an SP representative for the SP community; an ecosystem representative to interface ecosystem teams. This allows quicker feedback to the PL and FF. Representatives shall be rotated, and representatives shall be rotated quarterly or every 6 months.
6. **Increase the voting weight of storage power:** When voting, not only the weight is calculated according to the number of FIL in the wallet address, but also the proportion of storage power should be increased. The proportion of storage power in the voting weight should not be less than 30%. FIL holders and SPs provide the security and represent the most of the investment and takes most of the risks of the network, they should have higher say in governance.

**Comment by Fatman13** [comment]
> The proportion of storage power in the voting weight should not be less than 30%.

Lets make it 50% and the other 50% goes to token holders. 👍

**Reply by michael8908** [reply]
30%足矣，50%太高，SP一方独裁的局面，那就不是社区治理了。如果50%了，还要其他持币人投票方干嘛呢？与100%由SP投票决定，基本没有区别。

**Reply by Fatman13** [reply]
> 30%足矣，50%太高，SP一方独裁的局面，那就不是社区治理了。

矿工和持币人投票权重各占 50% 对 fil 社区最合理的，最公平的。假设，社区有人提了一个完全 favor 持币人的 FIP，最坏情况下，如按矿工持币人三七开权重，矿工全部投反对票，最终权重 30%，持币人全部投支持，最终权重 70%。在大多数矿工反对的情况下，按 70% 支持去通过这种 FIP。这才是你说的“单方面独裁的局面”。对网络健康影响非常不好。

**Reply by michael8908** [reply]
按你的说法，其实本质上就是完全由矿工决定嘛。50%的投票权，加上矿工本来就是持币大户，又何必还要其他群体投票呢？矿工投票就行了。同样也可以按你的说法来讲的：假如矿工提出了完全 favor 矿工的 FIP，结果 …… 。个人看法，持币人会更关心矿工的利益（相对矿工关心持币人的利益而言），毕竟持币的肯定不想矿工都不挖了，而矿工更关心的可能是“挖卖提”这件事，而不是长期持币。

**Comment by 852031971** [comment]
support

**Comment by fil88888888** [comment]
同意

**Comment by fil88888888** [comment]
支持

**Comment by kongxinputi** [comment]
support

**Comment by toutuoge** [comment]
支持

**Comment by jingyu0212** [comment]
支持！

**Comment by HJann762** [comment]
支持

**Comment by WANGDINGYIKECIN** [comment]
支持

**Comment by plutoyi** [comment]
支持

**Reply by nddiandao** [reply]
支持

**Comment by JIANGRUILIN** [comment]
支持！

**Reply by 205312327** [reply]
支持

**Reply by Yuehuajin1991** [reply]
希望在建设过程中的同时发挥FIL提升币价，吸引市场关注，只有体现Fil的价值，才会有更多人愿意了解它，有利于生态的繁荣。AR目前就火了

**Comment by sharkfee** [comment]
使用fil存储，解决垃圾数据问题

**Comment by kaitlin-beegle** [comment]
I want to comment on just the 'Proposal' elements.  I think a lot of these are great suggestions, and nearly all of these are things which are already in progress! 

> **Protocol Labs and Filecoin Foundation review the current tokenomics model**: Especially the designed mechanism of FIL+. Consult with the community and put forward new reform schemes to ensure the authenticity of Data Caps and promote the healthy development of verified data onboarding.

- I think this is a good suggestion! I agree that we should be really thoughtful about the role that subsidies play in our ecosystem.

> **Pay attention to market value management**: The healthy development of public chains needs more social attention, ecosystem developers, token holders, storage providers, investors and protocol engineers. In a competitive environment, low market value means that the prospects are slim and the pledge is devalued. Low market value destroys market confidence and keeps users away. Only a high market value can spur more investor’s attention. As a Web3.0 storage infrastructure, Filecoin should aim at the top ten in market value. Increase its ranking in crypto indexes such as CoinMarketCap.

- This is generally a good goal, but an increase in market cap only comes from increasing the true value of the ecosystem and protocol.  And fortunately, there is lots of good work being done to strengthen the Filecoin community and protocol stack.  For example, proposed FIPs that would make Filecoin [faster](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md) and [easier to use](https://github.com/filecoin-project/FIPs/pull/891), as well as [new working groups](https://github.com/filecoin-project/DeveloperWG) and [ecosystem partnerships](https://filecoin.io/blog/posts/filecoin-news-83/).

> **Strengthen the communication between the core team and the community:** The work of the core team determines the performance of the chain and the future development direction of the chain. Set up a joint working group, and the community and core team will hold regular meetings to inform the technical development planning and progress, incorporate feedback from the community on the pain points, and effectively advance the technology and meet market demands.

- There are lots of working groups that do this, including [Core Devs](https://github.com/filecoin-project/core-devs)! Are there other working groups that you'd like to propose? If so, maybe we could find a way to provide support for it! 

> **Return the governance back to the community:** Community voting system should be adopted for major issues. Major issues related to the ecosystem, such as the reform plan of Fil+ and the direction of technological development, are to be decided by community voting not by core member’s discussions.

Voting is a REALLY hard thing to do well, but we're [working on it](https://github.com/filecoin-project/FIPs/pull/850)! If you want to review the existing design spec for on-chain voting, you can do so [HERE](https://docs.google.com/document/d/13910NE-O3mUQ6rztt6f3xe7hwW_aS-xaPW_zHuTpBW4/edit#heading=h.4kbcnjlru68f).  We also host monthly [Community Governance Calls](https://github.com/filecoin-project/FIPs/tree/master/Community%20Governance%20Calls) and welcome engagement in the #fil-gov Slack channel!  If you're interested in contributing to this work then we'd love to have your help! :) 

Right now, Filecoin Foundation is working with the StorSwift team to build a test implementation of on-chain voting.  [Least Authority](https://leastauthority.com/) is scheduled to perform both a cryptoeconomic and code audit of the test tool, all of which will be made public in the next few months. 

> **Working groups rotate representatives/advocates**: For example, an SP representative for the SP community; an ecosystem representative to interface ecosystem teams. This allows quicker feedback to the PL and FF. Representatives shall be rotated, and representatives shall be rotated quarterly or every 6 months.

Yes!! This is also something that we have proposed in our [governance reform initiatives](https://github.com/filecoin-project/FIPs/pull/850)! I think this is a great suggestion. 

> **Increase the voting weight of storage power**: When voting, not only the weight is calculated according to the number of FIL in the wallet address, but also the proportion of storage power should be increased. The proportion of storage power in the voting weight should not be less than 30%. FIL holders and SPs provide the security and represent the most of the investment and takes most of the risks of the network, they should have higher say in governance.

In our current proposal, voting weight is calculated for the following categories: tokenholders (total token count), Core Developers (qualified contributions to core protocol development), Storage Providers (RBP), and clients (deal bytes). This is in an effort to prevent large exchanges or institutions (including Protocol Labs and Filecoin Foundation) from overwhelming community votes, as well as to reflect the varied ways in which community members contribute to building the Filecoin ecosystem. 

In this model, each 'category' of weight is capped at 25% of the overall outcome.  However, individual wallet addresses can cast votes in multiple categories.  This is a novel method that I have worked to develop and put forth several months ago, and I'd love any feedback that you may have.  

---

Ultimately, I am really excited about this proposal! I see a lot of SXX's suggestions in line with exactly the kinds of changes that many of us have been pushing for for several months! It is exciting to see community members care so much about the governance and development of this ecosystem :)

**Reply by ggyy2024** [reply]
很高兴看到kaitlin-beegle的回复。希望SXX和官方一起，让社区更有效率，更有活力

**Reply by sxxfuture-official** [reply]
@kaitlin-beegle 
> Pay attention to market value management: The healthy development of public chains needs more social attention, ecosystem developers, token holders, storage providers, investors and protocol engineers. In a competitive environment, low market value means that the prospects are slim and the pledge is devalued. Low market value destroys market confidence and keeps users away. Only a high market value can spur more investor’s attention. As a Web3.0 storage infrastructure, Filecoin should aim at the top ten in market value. Increase its ranking in crypto indexes such as CoinMarketCap.
- The increase in the true value of ecosystems and agreements is indeed the core factor of market value growth. We are very grateful to the foundation and technical partners for the excellent work. However, there is an obvious fact that the FIL held by the protocol labs and foundation accounts for a very large proportion of the total FIL released; And now, in the daily release, the number of protocol lab teams and foundations has exceeded all the storage incentives in the whole market. If the FIL held by the protocol labs and foundation can be properly placed and clearly explained, it will help the market confidence and boost the market value (especially if the FIL held by the protocol labs and foundation can better participate in the head project of intra-ecological liquidity pledge, it will greatly boost the confidence of all currency holders, contribute to the growth of the computing network, and better reduce the selling pressure, thus enhancing the market value in a more constructive way);

> Strengthen the communication between the core team and the community: The work of the core team determines the performance of the chain and the future development direction of the chain. Set up a joint working group, and the community and core team will hold regular meetings to inform the technical development planning and progress, incorporate feedback from the community on the pain points, and effectively advance the technology and meet market demands.
There are lots of working groups that do this, including Core Devs! Are there other working groups that you'd like to propose? If so, maybe we could find a way to provide support for it!
- We propose to set up a joint working group, including core developers, miners, investors and ecological project parties, and hold an online work exchange meeting once a month; At the same time, we propose that the foundation should set up a working group to coordinate and maintain the developers. This is not for centralization, but for improving the efficiency of work and communication.

> Return the governance back to the community: Community voting system should be adopted for major issues. Major issues related to the ecosystem, such as the reform plan of Fil+ and the direction of technological development, are to be decided by community voting not by core member’s discussions.
Voting is a REALLY hard thing to do well, but we're working on it! If you want to review the existing design spec for on-chain voting, you can do so HERE. We also host monthly Community Governance Calls and welcome engagement in the #fil-gov Slack channel! If you're interested in contributing to this work then we'd love to have your help! :)
Right now, Filecoin Foundation is working with the StorSwift team to build a test implementation of on-chain voting. Least Authority is scheduled to perform both a cryptoeconomic and code audit of the test tool, all of which will be made public in the next few months.
- Transparency and openness are the first principles of community construction. It is suggested to publicize seven lists of governance seats of Filecoin community association, how to select and establish these seven lists, and the governance process of community governance association.

> Increase the voting weight of storage power: When voting, not only the weight is calculated according to the number of FIL in the wallet address, but also the proportion of storage power should be increased. The proportion of storage power in the voting weight should not be less than 30%. FIL holders and SPs provide the security and represent the most of the investment and takes most of the risks of the network, they should have higher say in governance.
In our current proposal, voting weight is calculated for the following categories: tokenholders (total token count), Core Developers (qualified contributions to core protocol development), Storage Providers (RBP), and clients (deal bytes). This is in an effort to prevent large exchanges or institutions (including Protocol Labs and Filecoin Foundation) from overwhelming community votes, as well as to reflect the varied ways in which community members contribute to building the Filecoin ecosystem.
In this model, each 'category' of weight is capped at 25% of the overall outcome. However, individual wallet addresses can cast votes in multiple categories. This is a novel method that I have worked to develop and put forth several months ago, and I'd love any feedback that you may have.
- Thank you very much for your very effective work. Voting by role is a good solution, and we are looking forward to seeing it come true. The only thing that puzzles us is that 25% of customers' voting proportion can't be effective. From the perspective of effectiveness, because of the lack of language ability and proprietary network tools, the proportion of customers who have the ability to participate in decision-making is relatively low, and the implementation of 25% voting proportion has the problem of lack of governance. We suggest reducing the voting rate of customers to 20% and increasing the voting rate of storage providers to 30%. Because storage providers not only provide network infrastructure and services, customer communication is basically settled in these storage providers, which can effectively solve the language ability and special network tools. It is suggested that this part of the weight be added to storage providers.

**Comment by cj091116** [comment]
support

**Comment by xiaomdong** [comment]
support

**Comment by nddiandao** [comment]
support

**Comment by gugege521** [comment]
请官方能够多听取社区的声音，我们被币价折磨的非常难过，让fil回归社区治理吧，团队做好技术研发工作，带动fil币价重新起飞。

**Comment by piaoliuping1** [comment]
同意

**Comment by AlanGreaterheat** [comment]
Support！！！

**Comment by Bllaaa** [comment]
支持
